src/main/java/http/HttpRequest.java

```diff
@@ -1,29 +1,73 @@
 package http;
 
+import com.github.jknack.handlebars.internal.lang3.StringUtils;
+import http.enums.ContentType;
+import http.enums.Method;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import utils.IOUtils;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
 /**
  * Created By kjs4395 on 2020-06-05
  */
 public class HttpRequest {
-    private final RequestLine requestLine;
-    private final HttpHeaderInfo headerInfo;
-    private final String requestBody;
-
-    public HttpRequest(RequestLine requestLine, HttpHeaderInfo headerInfo, String requestBody) {
-        this.requestLine = requestLine;
-        this.headerInfo = headerInfo;
-        this.requestBody = requestBody;
+    private static final Logger logger = LoggerFactory.getLogger(HttpRequest.class);
+    private RequestLine requestLine;
+    private Header header;
+    private String requestBody;
+
+    public HttpRequest(InputStream in) {
+        try {
+            BufferedReader br = new BufferedReader(new InputStreamReader(in, "UTF-8"));
+            String line = br.readLine();
+            this.requestLine = RequestLineParser.parse(line.trim());
+            this.header = new Header();
+            this.requestBody = StringUtils.EMPTY;
+
+            while(!StringUtils.EMPTY.equals(line)) {
+                line = br.readLine().trim();
+                if(StringUtils.isNotEmpty(line)) {
```

Comment:
공백, 공백라인 컨벤션 등의 부분들은 습관적으로 반영하시는 것이 좋다고 생각합니다.
IntelliJ 를 사용하시는 경우 reformat 기능(```cmd(ctrl)``` + ```opt(alt)``` + ```L```)을 적극 사용하시길 권장드립니다. 😄 
```suggestion
            while (!StringUtils.EMPTY.equals(line)) {
                line = br.readLine().trim();
                if (StringUtils.isNotEmpty(line)) {
```